### ISSUE TITLE
No alternative text for close file preview button 

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#1534]
No alternative text on close file preview button

![screen shot 2017-01-24 at 15 09 10](https://cloud.githubusercontent.com/assets/13200338/22252814/4eeee4a8-e247-11e6-9572-ca55cf425db4.png)
![screen shot 2017-01-24 at 15 09 19](https://cloud.githubusercontent.com/assets/13200338/22252815/4ef75cf0-e247-11e6-888f-b672d79ee842.png)
